WordPress/Docs/PHP/StrictInArrayStandard.xml

```diff
@@ -0,0 +1,45 @@
+<documentation title="Strict In Array Syntax">
+    <standard>
+    <![CDATA[
+    Strict in array syntax must be used.
```

Comment:
Change `in array` to `in_array()` to make it clear we're talking about the function.

Reply:
Repeat for other instances throughout.